acsys/status.py

```diff
@@ -179,174 +216,209 @@ def create(val):
 
 # Clases that specialize Status - based on code, that can be captured on different except clauses
 class AcnetException(Status, Exception):
```

Comment:
I like this abstraction so you don't have to repeat `(Status, Exception)` a bunch.

Reply:
Yes, an excellent approach!

Reply:
That, and you need a way to capture all these exceptions, since Status is not an exception anymore.

Reply:
Well, the point was `Status` was to be success or warning level statues which no one would want to raise or catch. `except AcnetSuccess:` doesn't make sense.